test/common/upstream/outlier_detection_impl_test.cc

```diff
@@ -1915,6 +1915,44 @@ max_ejection_time_jitter: 13s
   EXPECT_FALSE(hosts_[2]->healthFlagGet(Host::HealthFlag::FAILED_OUTLIER_CHECK));
 }
 
+TEST_F(OutlierDetectorImplTest, MaxEjectionPercentageOverride) {
+  // Should eject one host even if mep doesn't allow it.
```

Comment:
Please add comment that the one host represents 33% which exceeds max_ejection_percent, which is 30. Please do not use abbreviations like "mep".